monai/utils/misc.py

```diff
@@ -217,3 +218,39 @@ def set_determinism(
         torch.backends.cudnn.benchmark = False
     else:
         torch.backends.cudnn.deterministic = False
+
+
+def list_to_dict(items):
```

Comment:
The `configparser` library might make sense for this but if the config is really simple it's just as well to parse ourselves.

Reply:
Hi @ericspod , thanks for your suggestion, we only use this utility to parse several key-value pairs in the config file.
If we have complicated objects to parse, we can try to use some 3rd party lib I think.
Thanks.